File: swagger/models/get_activation_code_response.py
from .base_model import BaseModel


class GetActivationCodeResponse(BaseModel):
	"""
	no description available

	Attributes:

		activation_code
		An activation code used to provide access to a site’s business data
		through MINDBODY.

		activation_link
		A link to the Manage Credentials screen.
	"""

	input_map = {
		'ActivationCode': 'activation_code',
		'ActivationLink': 'activation_link',
		}

	output_map = {
		'activation_code': 'ActivationCode',
		'activation_link': 'ActivationLink',
		}

File: swagger/Models/GetActivationCodeResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetActivationCodeResponse extends BaseModel
{
	/**
	 * An activation code used to provide access to a site’s business data
	 * through MINDBODY.
	 * 
	 * @var string 
	 */
	public $activationCode;
	/**
	 * A link to the Manage Credentials screen.
	 * 
	 * @var string 
	 */
	public $activationLink;

	protected function getInputMap()
	{
		return [
			'ActivationCode' => ['activationCode', 'string', null],
			'ActivationLink' => ['activationLink', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'activationCode' => ['ActivationCode', 'string', null],
			'activationLink' => ['ActivationLink', 'string', null],
			];
	}
}
